crates/rpc-types/src/eth/admin.rs

```diff
@@ -0,0 +1,409 @@
+//! Types for the admin api
+use alloy_genesis::ChainConfig;
+use alloy_primitives::{B256, U256};
+use alloy_serde::json_u256::deserialize_json_u256;
+use serde::{Deserialize, Serialize};
+use std::net::{IpAddr, SocketAddr};
+
+/// This includes general information about a running node, spanning networking and protocol
+/// details.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct NodeInfo {
```

Comment:
can we double check the field order of this type?

https://github.com/ethereum/go-ethereum/blob/767b00b0b514771a663f3362dd0310fc28d40c25/p2p/server.go#L1072-L1085

Reply:
Just checked, the order here is correct, should I fire a PR for reth?

https://github.com/paradigmxyz/reth/blob/0a803c49a9fac1bd44b94a1883cf352ced6816ee/crates/rpc/rpc-types/src/admin.rs#L16-L21

Reply:
not necessary, we'll replace reth's types with this